### PR TITLE
Add proper groups for Cody actions

### PR DESCRIPTION
**Pull request by pkukielka**
Closes https://linear.app/sourcegraph/issue/CODY-2095/organize-keymap-settings

## Changes

1. Proper grouping was added for all actions in plugin.xml
2. The same grouping is used for nice visual organisation of keymap management

## Test plan

Visual verification in `Settings > Keymap` (see screenshots):

image
image

### CODE REVIEW

**Comment by danielmarquespt** [comment]
@pkukielka Looks good! 

Small nit: Can you put the Sourcegraph group after Cody Accounts? Not a major thing

**Comment by pkukielka** [comment]
> @pkukielka Looks good!
> 
> Small nit: Can you put the Sourcegraph group after Cody Accounts? Not a major thing

Done!

File: src/main/java/com/sourcegraph/cody/CodyToolWindowFactory.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ public void createToolWindowContent(@NotNull Project project, @NotNull ToolWindo
   }
 
   private void createTitleActions(@NotNull List<? super AnAction> titleActions) {
-    AnAction action = ActionManager.getInstance().getAction("CodyChatActionsGroup");
```
Comment (mkondratek) [inline src/main/java/com/sourcegraph/cody/CodyToolWindowFactory.java:49]: before it was an action group, now it's an action - does it still work?

**Review by mkondratek — APPROVED** [review]

File: src/main/java/com/sourcegraph/cody/CodyToolWindowFactory.java (hunk 1 above)
Comment (pkukielka) [reply src/main/java/com/sourcegraph/cody/CodyToolWindowFactory.java:49]: I tested it before changing - it work just fine :)

**Pull request merged**